.github/workflows/image.yaml

```diff
@@ -22,8 +22,9 @@ jobs:
           go-version: 1.16
       - name: Build & Test
         run: |
-          go build -v ./cmd/controller
-          go test ./...
+          make toolchain
+          echo "/usr/local/kubebuilder/bin" >> $GITHUB_PATH
+          make build && make test
```

Comment:
I think you can say: `make toolchain build test`

Reply:
Nice! Thanks!